fitbenchmarking/fitbenchmarking
# Issue #213: Python3 compatibility

#### Description of Work

Ping #213 

This change makes all but mantid compatible with python 3.
Unfortunately Mantid wont be available until mid January.

I haven't tried to optimize anything for python3 as we need to support both until Mantid upgrades.


#### Testing Instructions

1. Check that it runs on python3 and python2

Function: Does the change do what it's supposed to?
Tests: Does it pass? Is there adequate coverage for new code?
Style: Is the coding style consistent? Is anything overly confusing?
Documentation: Is there a suitable change to documentation for this change?
